cloud/openstack/os_security_group.py

```diff
@@ -48,6 +48,8 @@
        - Should the resource be present or absent.
      choices: [present, absent]
      default: present
+
+requirements: ["shade"]
```

Comment:
if using docfragments that already have requirements, you don't need to add them again.